tutorials/Tutorial_For_QA_Registration.ipynb

```diff
@@ -0,0 +1,377 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# QA REGISTRATION\n",
+    "Give a visual result for regisration part to help user check results of each step. The graph includes three dimensions(sagital slice, coronal slice and axial slice) and uses pink and green to indicate the registered output and reference image."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup Steps\n",
+    "First we have to import all relevant functions used in this qa code. qa_reg uses ndmg functions from qa_utils and gen_utils."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/xueminzhu/anaconda3/lib/python3.7/site-packages/sklearn/externals/joblib/__init__.py:15: DeprecationWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.\n",
+      "  warnings.warn(msg, category=DeprecationWarning)\n"
+     ]
+    }
+   ],
+   "source": [
+    "import os\n",
+    "import nibabel as nb\n",
+    "from ndmg.utils.gen_utils import get_braindata, get_filename\n",
+    "from ndmg.utils.qa_utils import get_min_max, opaque_colorscale, pad_im\n",
+    "from argparse import ArgumentParser\n",
+    "from scipy import ndimage\n",
+    "from matplotlib.colors import LinearSegmentedColormap\n",
+    "from nilearn.plotting.edge_detect import _edge_map as edge_map\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Saving figures\n",
+    "Save the result returned by function plot_overlays and name the file as \"registered image \\_2\\_ reference image\"."
```

Comment:
Change "registered image_2_reference image" to "{registered image}_2_{reference image}"

Reply:
see https://github.com/neurodata/ndmg/pull/372/commits/7b32b3d114ee8d7a65060fb0dc1fefa8393cb052